src/index.mli

```diff
@@ -28,19 +28,20 @@ module type Key = sig
       drops. *)
 
   val hash_size : int
-  (** The maximum number of bits used to encode hashes. `Hashtbl.hash` uses 30
-      bits. *)
+  (** The number of bits necessary to encode the maximum output value of
+      {!hash}. `Hashtbl.hash` uses 30 bits. *)
```

Comment:
Maybe add a note on what happens if you use the wrong `size`? e.g. it's fine but slow if it's too large and it might crashes if it's too small.

Reply:
Done :heavy_check_mark: